### PR TITLE
refac(build): #320 support clojure linter

### PR DESCRIPTION
**Pull request by ludsrill**
- Add support to lint clojure code using clj-kondo

### CODE REVIEW

File: src/args/lint-clojure/builder.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# shellcheck shell=bash
+
+function main {
+  local args=(
+    --lint
+  )
+
+  info Compiling Clojure solution \
```
Comment (kamadorueda) [inline src/args/lint-clojure/builder.sh:8]:
```suggestion
  info Linting Clojure solution \
```

File: src/args/lint-clojure/builder.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    && clj-kondo "${args[@]}" "${envTarget}" \
+    || critical Unsuccessful compilation \
```
Comment (kamadorueda) [inline src/args/lint-clojure/builder.sh:10]:
```suggestion
```

we can add these messages at the CLI level, let's avoid adding them here

File: README.md [hunk 3]
```diff
@@ -756,6 +757,38 @@ Example `makes.nix`:
...
+      ];
+      example2 = [
+        "/directory" # A directory within the project
+      ];
+    };
+  };
+}
+```
+
+Example invocation: `$ m . /lintClojure/example1`
```
Comment (kamadorueda) [inline README.md:788]: you are not using `targets`, so we can remove it

```nix
{
  lintClojure = {
      example1 = [
        "/" # Entire project
        "/file.py" # A file
      ];
      example2 = [
        "/directory" # A directory within the project
      ];
    };
}
```

**Review by kamadorueda — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (ludsrill) [reply README.md:788]: same for lintWithLizard?

File: src/args/lint-clojure/builder.sh (hunk 2 above)
Comment (ludsrill) [reply src/args/lint-clojure/builder.sh:10]: you mean add this to shell-commands, right?

Comment (kamadorueda) [reply src/args/lint-clojure/builder.sh:10]: no, to src/cli/main/__main__.py

in other words just remove the line, I'll add the 'success/fail' message later

File: README.md (hunk 3 above)
Comment (kamadorueda) [reply README.md:788]: would be a nice idea for simplicity

Comment (ludsrill) [reply README.md:788]: on it

File: README.md [hunk 4]
```diff
@@ -756,6 +757,36 @@ Example `makes.nix`:
 
 Example invocation: `$ m . /lintBash`
 
+### lintClojure
+
+Lints clojure code with [clj-kondo][CLJ-KONDO].
+
+Attributes:
+
+- targets (`attrsOf (listOf str)`): Optional.
```
Comment (kamadorueda) [inline README.md:766]:
```suggestion
- self (`attrsOf (listOf str)`): Optional.
```

File: README.md [hunk 5]
```diff
@@ -756,6 +757,36 @@ Example `makes.nix`:
...
+  Defaults to `{ }`.
+
+Example `makes.nix`:
+
+```nix
+{
+  lintClojure = {
+    example1 = [
+      "/" # Entire project
+      "/file.py" # A file
```
Comment (kamadorueda) [inline README.md:777]:
```suggestion
      "/file.clj" # A file
```

**Review by kamadorueda — CHANGES_REQUESTED** [review]

**Pull request closed without merging**